rna/update-samples-entity/update-samples-entity-from-samples-list.terra.inputs.json

```diff
@@ -0,0 +1,6 @@
+{
+  "SampleToPairedFastq.mapping_file": "${}",
```

Comment:
Matt, I left this purposefully blank. What do you think?

Reply:
That seems right to me. Can you have the comment say "GCS Path to the mapping tsv file"?